src/Core/Settings.cpp

```diff
@@ -4312,14 +4312,14 @@ SELECT * FROM x_dist ORDER BY number ASC;
     DECLARE(Bool, collect_hash_table_stats_during_aggregation, true, R"(
 Enable collecting hash table statistics to optimize memory allocation
 )", 0) \
-    DECLARE(UInt64, max_size_to_preallocate_for_aggregation, 100'000'000, R"(
+    DECLARE(UInt64, max_size_to_preallocate_for_aggregation, 1'000'000'000'000, R"(
```

Comment:
Do we really target for a trillion, or use this as a "big enough" value?

Reply:
the latter. you think i should change it to `std::numeric_limits<UInt64>::max()`?

Reply:
This is ok, keep it :)